bolt-modules/boltlib/lib/puppet/datatypes/target.rb

```diff
@@ -4,16 +4,16 @@
   begin
     inventory = Puppet.lookup(:bolt_inventory)
 
-    inventory_version = inventory.version
-    if inventory_version != 1
+    inventory_name = inventory.name
```

Comment:
I'm not sure what this buys us, since it looks like the only place we check for `v2` is in the CLI (where `apply` is known to not be possible) and it doesn't look like we check for `apply` anywhere. I was thinking of the version as the API version that the inventory conforms to (1 or 2) rather than an identifier about which inventory it is.

Reply:
Hm that's a good point - now that we have an apply inventory object with the target_implementation_class method this isn't necessary.